notifications_api_common/viewsets.py

```diff
@@ -162,46 +169,39 @@ def notify(
         # build the content of the notification
         message = self.construct_message(data, instance=instance)
 
+        self.send_notification.delay(status_code, message)
+
+    @staticmethod
+    @celery.current_app.task(bind=True)
+    def send_notification(
+        task,
+        status_code: int,
+        message: dict,
+    ) -> None:
         # build the client from the singleton config. This will raise an
         # exception if the config is not complete. We want this to hard-fail!
         client = NotificationsConfig.get_client()
         if client is None:
             raise RuntimeError("Could not build a client for Notifications API")
```

Comment:
this is a behaviour change - if the notifications API is not properly configured yet, now the API endpoint transaction will go through rather than abort and no notification can be sent.